src/components/DeveloperCard.tsx

```diff
@@ -37,4 +37,4 @@ export default function DeveloperCard(props: DeveloperProps){
             </div>
         </div>
     )   
-}
\ No newline at end of file
+}
```

Comment:
I just wanna say that I **_love_** the fact that you included a blank line at the end of the files.

Reply:
:joy: It's not mandatory, but if you use any type of `Linter` it'll force you to do it. And Github alerts you that the blank line is missing, so...

Reply:
Yes yes yes yes, let us let Linter take care of our code please